app/views/teams/_form.html.erb

```diff
@@ -26,11 +26,6 @@
     <%= form.check_box :is_parent %>
   </div>
 
-  <div class="field">
```

Comment:
of course you could get fancy and have an "activate upon creation" checkbox, but then you'd have to have future activation ui stuff. just trying to be difficult ;-P